docs/source/getting-started/tracking-server-overview/index.rst

```diff
@@ -0,0 +1,327 @@
+.. _tracking-server-overview:
+
+5 Minute Tracking Server Overview
+======================================================================
```

Comment:
The header length can extend to the length of the preceding line (just for consistency). It's fine if it's longer to sphinx (just not shorter), but my control-freak nature twitches when I see these extend beyond the header text ;)